plugins/module_utils/ansible_freeipa_module.py

```diff
@@ -282,6 +282,11 @@ def compare_args_ipa(module, args, ipa):  # noqa
     """
     base_debug_msg = "Ansible arguments and IPA commands differed. "
 
+    if args is None and ipa is None:
```

Comment:
Please add a comment what this is fixing and when this applies.

Reply:
done.